squad/core/management/commands/update_project_statuses.py

```diff
@@ -0,0 +1,30 @@
+from datetime import timedelta
+from dateutil import parser as date_parser
+from django.core.management.base import BaseCommand
+from django.utils import timezone
+
+from squad.core.models import ProjectStatus, Build
+
+
+class Command(BaseCommand):
+
+    help = """Update project status records in order to fix a bug which compared current builds with the wrong ones."""
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--date-start',
+            dest="date_start",
+            default=(timezone.now() - timedelta(days=180)),
+            type=date_parser.isoparse,
```

Comment:
I'm getting the following error here:
```
Traceback (most recent call last):
  File "./manage.py", line 5, in <module>
    main()
  File "/home/milosz/linaro/tcstorm/squad/squad/manage.py", line 14, in main
    execute_from_command_line(sys.argv)
  File "/home/milosz/linaro/tcstorm/squad_venv/lib/python3.5/site-packages/django/core/management/__init__.py", line 364, in execute_from_command_line
    utility.execute()
  File "/home/milosz/linaro/tcstorm/squad_venv/lib/python3.5/site-packages/django/core/management/__init__.py", line 348, in execute
    self.fetch_command(options.args[0]).print_help(self.prog_name, options.args[0])
  File "/home/milosz/linaro/tcstorm/squad_venv/lib/python3.5/site-packages/django/core/management/base.py", line 263, in print_help
    parser = self.create_parser(prog_name, subcommand)
  File "/home/milosz/linaro/tcstorm/squad_venv/lib/python3.5/site-packages/django/core/management/base.py", line 249, in create_parser
    self.add_arguments(parser)
  File "/home/milosz/linaro/tcstorm/squad/squad/core/management/commands/update_project_statuses.py", line 18, in add_arguments
    type=date_parser.isoparse,
AttributeError: module 'dateutil.parser' has no attribute 'isoparse'
```

This is on python 3.5.2. Isn't dateutil 3rd party package? If so we're missing a dependency.

Reply:
Also I'm missing the explanation about the date format. I tried YYYY.MM.DD and it didn't work. YYYY-MM-DD works.

Reply:
> Also I'm missing the explanation about the date format. I tried YYYY.MM.DD and it didn't work. YYYY-MM-DD works.

There's already a format in the help text, where else do you think should we should add the description?

Reply:
That works. Is there a way to display help text on input parsing error?

Reply:
It already does, I think the only problem here is the missing dependency.

Reply:
I'm not sure what's going on here. There's already python_dateutil in requirements.txt and mention of it in test/ci/test_models.py. Strange that it doesn't work for you, can you check your dateutil version with "pip show python_dateutil" ?

Reply:
yes but we need to know what is the minimum version of dateutil that has what we need

Reply:
@terceiro so should I put the minimum version to 2.7.0 in requirements.txt (and the dev one)?

Reply:
@stevanradakovic now I'm not sure. the rest of squad works fine with an older dateutil, maybe we don't want to enforce such requirement since dateutil 2.7 is relatively recent

Reply:
Ok that sounds fair. I can find some other way to do command line argument validation.